app/src/main/java/com/infomaniak/mail/ui/main/MainViewModel.kt

```diff
@@ -167,7 +167,7 @@ class MainViewModel : ViewModel() {
         val mailboxUuid = MailboxController.getMailboxSync(mailboxObjectId)?.uuid ?: return@launch
         val folderId = currentFolderId.value ?: return@launch
         currentOffset = OFFSET_FIRST_PAGE
-        isDownloadingChanges = true
+        withContext(Dispatchers.Main) { isDownloadingChanges.value = true }
```

Comment:
It's a `postValue` that needs here instead of `value` and `withContext(Dispatchers.Main)`